treegp/utils.py

```diff
@@ -0,0 +1,98 @@
+import numpy as np
+
+# TO DO: E/B mode computation using numpy should use
+# treecorr instead.
+
+
+def vcorr(x, y, dx, dy, rmin=5.0 / 3600.0, rmax=1.5, dlogr=0.05, maxpts=30000):
```

Comment:
It would be helpful to note that `rmin` and `rmax` are in degrees.

Reply:
Just did!